src/script/assets/AssetCrypto.ts

```diff
@@ -28,51 +34,32 @@ export const decryptAesAsset = async (
   keyBytes: ArrayBuffer,
   referenceSha256: ArrayBuffer,
 ): Promise<ArrayBuffer> => {
-  return window.crypto.subtle
-    .digest('SHA-256', cipherText)
-    .then(computedSha256 => {
-      if (_equalHashes(referenceSha256, computedSha256)) {
-        return window.crypto.subtle.importKey('raw', keyBytes, 'AES-CBC', false, ['decrypt']);
-      }
-
-      throw new Error('Encrypted asset does not match its SHA-256 hash');
-    })
-    .then(key => {
-      const iv = cipherText.slice(0, 16);
-      const assetCipherText = cipherText.slice(16);
-      return window.crypto.subtle.decrypt({iv: iv, name: 'AES-CBC'}, key, assetCipherText);
-    });
+  const computedSha256 = await window.crypto.subtle.digest('SHA-256', cipherText);
+  if (!_equalHashes(referenceSha256, computedSha256)) {
+    throw new Error('Encrypted asset does not match its SHA-256 hash');
+  }
+  const key = await window.crypto.subtle.importKey('raw', keyBytes, 'AES-CBC', false, ['decrypt']);
+  const iv = cipherText.slice(0, 16);
```

Comment:
Is there a more descriptive name for `iv`?

Reply:
Yes, actually it's an initialization vector.